src/screens/Ticketing/Ticket/TicketInfo.tsx

```diff
@@ -150,16 +147,6 @@ const TicketInfoTexts = (props: TicketInfoViewProps) => {
           {tariffZoneSummary}
         </ThemeText>
       )}
-      {isInspectable === false &&
-        status !== 'expired' &&
-        status !== 'refunded' && (
-          <View style={styles.notInspectableWarning}>
-            <ThemeIcon svg={Warning} style={styles.notInspectableWarningIcon} />
-            <ThemeText isMarkdown={true}>
-              {t(TicketTexts.ticketInfo.notInspectableWarning)}
-            </ThemeText>
-          </View>
-        )}
```

Comment:
Hvordan blir det for de som ikke er innrullert i mobiletoken, men har t:kort på profilen sin i Firestore? De er vel forventet at fortsatt skal få denne meldingen, slik som før 🤔

Reply:
det er et veldig bra poeng, og må fikses